### PR TITLE
Initial no-ship delivery of HTTP Stats (for Mbeans and for mpMetrics-5.x)

### PR DESCRIPTION
**Pull request by Channyboy**
#build

### CODE REVIEW

**Comment by Channyboy** [comment]
Addressed comments from the other PR https://github.com/OpenLiberty/open-liberty/pull/28406 (this branch is properly named).
@fmhwong 
#build #libby

File: dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+import jakarta.servlet.http.HttpServletResponse;
+
+/**
+ *
+ */
+public class ServletFilter implements Filter {
+	
+	private static final TraceComponent tc = Tr.register(ServletFilter.class);
+	
+	private static final String REST_HTP_ROUTE_ATTR = "RESTFUL.HTTP.ROUTE";
```
Comment (fmhwong) [inline dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java]: `REST_HTTP_ROUTE_ATTR` or `RESTFUL_HTTP_ROUTE_ATTR`?

Comment (Channyboy) [reply dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java]: I guess the value should be  REST.HTTP.ROUTE

**Comment by Channyboy** [comment]
#build #libby

**Review by fmhwong — DISMISSED** [review]
LGTM

**Comment by jim-krueger** [comment]
I assume there are (or will be) FAT tests added for this?   Please include the issue/pr number for that here.

**Review by jim-krueger — DISMISSED** [review]
Approved, but needs link to corresponding FAT issue/PR.

**Comment by Channyboy** [comment]
#build #libby

**Review by fmhwong — APPROVED** [review]
LGTM

**Pull request merged**